core/src/main/scala/com/fortysevendeg/mvessel/Connection.scala

```diff
@@ -7,14 +7,13 @@ import java.util.concurrent.Executor
 import com.fortysevendeg.mvessel.logging.{AndroidLogWrapper, LogWrapper}
 import com.fortysevendeg.mvessel.metadata.DatabaseMetaData
 import com.fortysevendeg.mvessel.statement.{PreparedStatement, Statement}
-
-import scala.util.{Failure, Success, Try}
+import com.fortysevendeg.mvessel.util.DatabaseUtils._
 
 class Connection(
```

Comment:
Yeah, my only concern is that the `Connection` class is an implementation for the java.sql.Connection. I think it's better to let a pure class. thanks